lib/valid_email/ban_partial_disposable_email_validator.rb

```diff
@@ -0,0 +1,23 @@
+require 'active_model'
+require 'active_model/validations'
+require 'valid_email/validate_email'
+class BanPartialDisposableEmailValidator < ActiveModel::EachValidator
+  # A list of disposable email domains
+  def self.config=(options)
+    @@config = options
+  end
+
+  # Required to use config outside
+  def self.config
+    @@config = [] unless defined? @@config
+
+    @@config
+  end
```

Comment:
Why is the configuration inside the validator? I think it should go to a configuration for the overall ValidEmail gem